simulation/halsim_xrp/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+
+cc_library(
+    name = "halsim_xrp",
+    srcs = glob([
+        "src/main/native/cpp/*",
+        "src/main/native/include/*.h",
+    ]),
+    features = ["windows_export_all_symbols"],
```

Comment:
This smells like it deserves to be in a macro, and needs a comment.  What are you trying to achieve with this?

Reply:
I'll remove it for now. At the moment the main repo doesn't build `.dll's` on windows. For the most part symbols aren't explicitly exported in allwpilib, so you need to tell bazel to generate the appropriate def files